### ISSUE TITLE
JAWS does not support `aria-label` in combination with `role='form'` on `div`

### ISSUE THREAD

**Issue by xmicore** [FreedomScientific/standards-support#748]
### Summary

A `div` element described by the attribute `aria-label` in combination with `role='form'` is not announced.

The W3C list `aria-label` under _Inherited States and Properties_:
https://w3c.github.io/aria/#form

MDN show the scenario as the first code snippet on the reference for ARIA form role:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/form_role

````
<div role="form" id="contact-info" aria-label="Contact information">
  <!-- form content -->
</div>
````

### Expected result

The `div` is announced by the value of `aria-label`.

### Actual result

JAWS announce only the content of the `div` element.

### Example

https://codepen.io/xmicore/pen/MWZKdQm

#### JAWS version and build number

JAWS Prefessional Edition 2021.2012.48 ILM
 
#### Operating System and version

Windwos 10

#### Browser and version:

Google Chrome 116.0.5845.111
Mozilla Firefox 102.14.0esr

**Comment by stevefaulkner** [comment]
tested with JAWS 2023.2307.37 and Chrome Version 116.0.5845.112 (Official Build) (64-bit) using this test case https://codepen.io/stevef/pen/mdaPjvr

aria-label content announced and form region recognised/announced

**Comment by xmicore** [comment]
@stevefaulkner Can you test the issue with Firefox?

**Comment by stevefaulkner** [comment]
> @stevefaulkner Can you test the issue with Firefox?

works fine in Firefox 117.0 (64-bit)
